src/main/webapp/app/core/login/login.service.ts

```diff
@@ -34,6 +40,7 @@ export class LoginService {
     }
 
     logout() {
+        this.localStorageService.clear();
```

Comment:
What would this clear? The whole localStorage?
If this is the case, this is not a good idea because the localStorage is supposed to be kept after the login.

Atm we e.g. use it for the code-editor to persist the error annotations in the editor.